labelbox/schema/project.py

```diff
@@ -105,10 +107,9 @@ def update(self, **kwargs):
                 auto_audit_percentage <= 1.0
             Attempting to switch between benchmark and consensus modes is an invalid operation and will result
             in an error.
+
+        For more information, visit https://docs.labelbox.com/reference/migrating-to-workflows#upcoming-changes
```

Comment:
like this info! wondering if instead of in project.update(), we ought to just include this in the changelog? 🤔